joomla/joomla-cms
# Fix modal in outdated overrides

Pull Request for Issue #23199.

### Summary of Changes

Fixes consent field modal in old overrides.

### Testing Instructions

Set Beez3 as default template.
Enable `Content - Confirm Consent` plugin and assign a privacy article.
Go to a contact form.
Click on `Privacy Note`.

### Expected result

Modal with privacy article opens.

### Actual result

Nothing happens.

### Documentation Changes Required
IDK.
